distribution/changelog.txt

```diff
@@ -1,6 +1,7 @@
 0.4.7 (in development)
 ------------------------------------------------------------------------
 - Fix: [#20737] Spent money in player window underflows when getting refunds.
+- Fix: [#20747] Handle staff speed permanency/serialization/apply for new staff.
```

Comment:
```suggestion
- Fix: [#20747] Staff speed cheat not applying to newly hired staff, UI not showing the current applied speed.
```
The changelog should say what it fixes.

Reply:
I'm currently on holidays oversees with no laptop. Will apply the fixes in approx 2weeeks.